demos/crossroad_camera_demo/main.cpp

```diff
@@ -23,6 +23,7 @@
 
 #include <inference_engine.hpp>
 
+#include <images_capture.h>
```

Comment:
I think this should be in the `samples` directory, to align it with the rest of the headers.

Reply:
I think `samples` was kept just to avoid updating every demo with removing `samples` from includes. But I moved `images_capture.h` to align headers for now.

Reply:
> I think samples was kept just to avoid updating every demo with removing samples from includes.

That _is_ why I kept it, although I don't think it should be removed entirely - it should just be renamed to `common` to match the name of the library. (Not in this PR, obviously.)